modules/ROOT/nav.adoc

```diff
@@ -243,7 +243,7 @@
   *** xref:fts:fts-architecture-scatter-gather.adoc[Scatter Gather]
   *** xref:fts:fts-architecture-ports-used.adoc[Ports Used By FTS]
   *** xref:fts:fts-rebalance-failover.adoc[Rebalance/Failover]
- ** xref:fts:fts-cluster-operations.adoc[Cluster Level Options]
+ ** xref:fts:fts-cluster-options.adoc[Cluster Level Options]
```

Comment:
If you're doing this, you'll be breaking the link without updating the adoc from fts-cluster-operations to fts-cluster-options

Reply:
Ah nevermind, I was on an older fork of release/7.0.